applications/openshift/api-server/api_server_audit_log_maxage/rule.yml

```diff
@@ -1,32 +0,0 @@
-documentation_complete: true
-
-prodtype: ocp4
-
-title: 'Configure the Audit Log Maximum Retention Days (maximumFileRetentionDays)'
```

Comment:
@JAORMX you are still going to need this rule in addition to offloading audit logs. Security conscious environments like to have this configured in case offloading fails for some reason (which does happen). Loss of audit logs is a bad thing.

Reply:
there is a hardcode default for that already. Didn't think it was necessary given that.

Reply:
ahh... got it. Thanks.